src/map/battle.cpp

```diff
@@ -9490,8 +9478,8 @@ int64 battle_calc_return_damage(struct block_list* tbl, struct block_list *src,
 		rdamage = battle_calc_gvg_damage(src, tbl, rdamage, skill_id, flag);
 	else if (mapdata->getMapFlag(MF_BATTLEGROUND))
 		rdamage = battle_calc_bg_damage(src, tbl, rdamage, skill_id, flag);
-	else if (mapdata->getMapFlag(MF_PVP))
-		rdamage = battle_calc_pk_damage(*src, *tbl, rdamage, skill_id, flag);
+	else if (battle_config.pk_mode > 0 && mapdata->getMapFlag(MF_PVP))
+			damage = battle_calc_pk_damage(*src, *tbl, damage, skill_id, flag);
```

Comment:
```suggestion
		damage = battle_calc_pk_damage(*src, *tbl, damage, skill_id, flag);
```
Why the change from rdamage to damage?